indexer-app/src/cmr/indexer/services/index_service.clj

```diff
@@ -663,15 +663,24 @@
   "Performs the cascade actions of collection deletion,
   i.e. propagate collection deletion to granules and variables"
   [context concept-mapping-types concept-id revision-id]
-  (doseq [index (idx-set/get-granule-index-names-for-collection context concept-id)]
-    (let [resp (es/delete-by-query
-                context
-                index
-                (concept-mapping-types :granule)
-                {:term {(query-field->elastic-field :collection-concept-id :granule)
-                        concept-id}})]
-      (when (not= (get resp :status) 200)
-        (warn (format "cascade collection delete for concept id %s and revision id %s did not return 200 status response. Elastic delete by query resp = %s" concept-id revision-id resp)))))
+  (let [small-collections-index-name (:small_collections (:granule (:index-names (idx-set/get-concept-type-index-names context))))]
```

Comment:
this needs threaded

Reply:
For just the let variable, or a larger section?

Reply:
Just this binding.